CONTRIBUTING.md

```diff
@@ -226,17 +226,17 @@ Changes should be atomic; do not combine multiple, discrete changes within a sin
 We use [squash merge](https://docs.github.com/en/enterprise-cloud@latest/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#merge-message-for-a-squash-merge) to create a single fresh commit based on PR title. The PR title should follow [Conventional Commit guidelines](https://www.conventionalcommits.org/en/v1.0.0/). Acceptable "types" are:
 
 * **doc**: purely documentation updates (semver patch)
-  * "scope" is usually the loader's name
+  * "scope" is usually the loader's name. ex `doc(tsx): update …`, `doc(tsx,yml): update …`
 * **dep**: a dependency update (semver patch or major)
-  * "scope" is the dependency's name
+  * "scope" is the loader's name. ex `dep(tsx): update DEP_NAME`, `dep(tsx,yml): update DEP_NAME`
```

Comment:
is dependabot can do that ?
Maybe use group ?

Reply:
Dependabot does do that :) I configured it to do. But if dependabot is doing it wrong, we want to know so we can fix it.

> Maybe use group ?

Hmm? What do you mean "use group"?

Reply:
we use group for each loader and setup prefix as `dip(loader):`

Reply:
I assume by `dip` you meant `dep`?

We want to know _which_ loaders are affected though, for when cutting the release.

Reply:
> I assume by dip you meant dep?

Reply:
I misspoke, with my proposal you'll have the same rendering but it's simpler.

Reply:
i assume yeah